app/presenters/ratings_presenter.rb

```diff
@@ -0,0 +1,4 @@
+class RatingsPresenter < Jsonite
+  property :rating
+  property :review, with: ReviewsPresenter
```

Comment:
In [my PR for ratings](https://github.com/ShelterTechSF/askdarcel-api/pull/107/files#diff-b876217aacfcd1aa62c2a771de35f8b2R4) I flattened this out, which seems a bit more straightforward, since a review is just a string. We can drop the extra presenter if we go this way.